cli/cmd/plugin/ui/web/tanzu-ui/src/views/management-cluster/aws/aws-mc-common/aws-orchestrator/AwsOrchestrator.service.tsx

```diff
@@ -87,4 +89,64 @@ export class AwsOrchestrator {
             setErrorObject(addErrorInfo(errorObject, e, AWS_FIELDS.NODE_TYPE));
         }
     }
+
+    static initAvailabilityZones(props: AwsOrchestratorProps) {
+        const { awsDispatch, setErrorObject, errorObject } = props;
+        DefaultOrchestrator.initResources<AWSKeyPair>({
```

Comment:
I think the type here should be `AWSAvailabilityZone` rather than `AWSKeyPair`?

Reply:
Yes sir.